src/main/java/aeneas/models/LevelIndex.java

```diff
@@ -19,13 +19,12 @@
 public class LevelIndex {
   HashMap<Integer, Level> levels;
 
-  public final String defaultLevelPath;
+  public final Path defaultLevelPath;
 
   LevelIndex() {
     levels = new HashMap<>();
     // TODO: Consider changing this path.
-    String homeDir = System.getenv("HOME");
-    defaultLevelPath = homeDir + "/.aeneas-kabasuji";
+    defaultLevelPath = Paths.get(System.getenv("HOME"), ".aeneas-kabasuji");
```

Comment:
Looks like `System.getProperty("user.home");` might work for all platforms. We should test it to be sure though.